qa/core-application-e2e-test-suite/fixtures.ts

```diff
@@ -27,6 +31,8 @@ type PlaywrightFixtures = {
   operateProcessInstancePage: OperateProcessInstancePage;
   taskDetailsPage: TaskDetailsPage;
   tasklistHeader: TasklistHeader;
+  tasklistprocessesPage: TasklistProcessesPage;
```

Comment:
I guess you'll fix it on the other PR

```suggestion
  tasklistProcessesPage: TasklistProcessesPage;
```